### PR TITLE
MAT-7206: Implement workflow for bad cql library name

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7206](https://jira.cms.gov/browse/MAT-7206)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/measureLanding/measureList/InvalidMeasureNameDialog/getLibraryNameErrors.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  qdmCheck: {
+    test: (name: string, model: Model) => {
+      if (model !== Model.QDM_5_6) {
+        return true;
+      }
+      return /^[A-Z][a-zA-Z0-9_]*$/.test(name);
+    },
+    message:
+      "Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters except of underscore for QDM.",
```
Comment (jkotanchik-SB) [inline src/components/measureLanding/measureList/InvalidMeasureNameDialog/getLibraryNameErrors.ts]: I remember adjusting this language elsewhere... 🤔 That happened, right?
```suggestion
      "Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters except for underscore in the case of QDM.",
```

**Review by jkotanchik-SB — DISMISSED** [review]

**Review by sb-prateekkeerthi — APPROVED** [review]
